web-ui/src/compositions/ui/pageTitle.ts

```diff
@@ -0,0 +1,12 @@
+import type { ReactNode } from 'react'
```

Comment:
maybe this can be under `components/page-header` rather than new folder?

Reply:
as a side note: we should probably come up with some guideline on how to structure the folders/place files.

Reply:
I'd like to leave it here as it doesn't really below to components/ , and have a separate discussion on dir structure